generators/client/templates/angular/src/main/webapp/app/app.main.ts.ejs

```diff
@@ -16,6 +16,10 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
+import 'core-js/es6/reflect';
+import 'core-js/es7/reflect';
+import 'zone.js/dist/zone';
```

Comment:
shouldn't `zone` and below manifest import still be part of polyfills?

Reply:
No, they are required for Angular in JIT mode for the `reflect` imports, and Zone is always required.
Otherwise you will get a very strange `cannot find Parameter for ApplicationModule(?)` 😄

Reply:
I am bit confused on mixing pollyfills with application code. If some polyfills are only required in JIT mode like `reflect`, then, can we separate those out and exclude from AOT builds or would AOT compiler be excluding those by default?

Thanks for reminding, I know angular is leveraging `zone` as base construct to support asynchronous events

Reply:
Yes maybe we can separate this into a `required-polyfills` file ?
The AOT won't exclude those files automatically

Reply:
I like the angular CLI approach and naming convention to move those into specific `es2015-polyfills.ts` and let the main be `polyfills.ts`

`JIT` mode specific polyfills are moved to separate `es2015-jit-polyfills.ts` and included only in JIT compilation (to save some kbs)

Reply:
That's good idea but I find it a little overkill just for to save two files which are not huge.. I prefer keeping it simple, even if we bundle them in AOT mode which is useless.

I think we will pretty soon use AOT even for dev so we will remove those polyfills once we've done that :)

Reply:
agree, effort may not be worth to save few kbs